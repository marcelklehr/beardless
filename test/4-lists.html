<ul><li data-template="linklist"></li></ul>