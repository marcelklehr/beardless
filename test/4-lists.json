{ "linklist":
  [ "Hello world"
  , "It never sleeps"
  , "It just idles"
  ]
}