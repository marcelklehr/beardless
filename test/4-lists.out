<ul><li>Hello world</li><li>It never sleeps</li><li>It just idles</li></ul>